chore(http): Refresh headers ~[s->x] (#36862)
mdn/content: files/en-us/glossary/source_map/index.md

---
title: Source map
slug: Glossary/Source_map
page-type: glossary-definition
---

{{GlossarySidebar}}

> A **source map** is a file that maps transformed code back to a source, enabling the browser to reconstruct the original source code and show that reconstructed code in the debugger.

Reviewer: This is messy too. I'm not sure how to fix this  - I like a succinct intro sentence like this, but this is hard to do "meaningfully" without the explanation that follows.

Maybe

```suggestion
A **source map** is a file that maps between minified and/or combined code received by the browser and its original unmodified form, allowing the original code to be reconstructed and used when debugging.
```

Author: Agree, it's hard without the background, but I like your version better, tnx

Author: done in f20875ebd209e00620978ab092d3a6ce6662e9ec

> The JavaScript sources executed by the browser are often transformed in some way from the sources created by a developer.

Author: ```suggestion
The JavaScript code executed by the browser is usually transformed in some way from the source created by a developer.
```

Reviewer: Good catch. Let's change the tense.
```suggestion
The JavaScript code executed by the browser has often been transformed in some way from the original source created by a developer.
```
